src/NRedisStack/Search/Literals/FieldOptions.cs

```diff
@@ -1,11 +1,13 @@
 ﻿namespace NRedisStack.Search.Literals;
 
-internal class AttributeOptions
+public class FieldOptions
```

Comment:
It seems you're doing this only for test purpose. If so, there should be a comment stating that.

Reply:
what do you mean by "only for test purpose" ?

Reply:
It seems you made this FieldOptions (AttributeOptions) public, so that it can be used in TestMissingEmptyFieldCommandArgs test. That's what I meant by "only for test purpose".

Reply:
ohhh i see, good catch.
It was not my intention TBH.